src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/helper/LetterAddressHelperTest.java

```diff
@@ -0,0 +1,45 @@
+package uk.gov.hmcts.reform.finrem.caseorchestration.helper;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.Test;
+import uk.gov.hmcts.reform.ccd.client.model.CallbackRequest;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+
+import java.io.InputStream;
+import java.util.Map;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+import static uk.gov.hmcts.reform.finrem.caseorchestration.OrchestrationConstants.LINE_SEPARATOR;
+
+public class LetterAddressHelperTest {
+
+    private ObjectMapper mapper = new ObjectMapper();
+
+    private LetterAddressHelper letterAddressHelper = new LetterAddressHelper();
+
+    private CaseDetails caseDetails() throws Exception {
```

Comment:
in my opinion you don't have to load values from a file. You can just create them as a map of value for each test case.